integration/bundle/init_default_python_test.go

```diff
@@ -59,6 +59,9 @@ func testDefaultPython(t *testing.T, pythonVersion string) {
 	require.NotNil(t, user)
 	testdiff.PrepareReplacementsUser(t, replacements, *user)
 	testdiff.PrepareReplacements(t, replacements, wt.W)
```

Comment:
nit: probably should also do PrepareReplacements -> PrepareReplacementsWorkspaceClient

Reply:
Done.